src/pages/index.tsx

```diff
@@ -54,11 +54,11 @@ const Home: NextPage = () => {
         verticalAlignment="center"
       >
         <LogoIcon color="white" size="display" />
-        <Display color="white" align="center">
-          Turn Surveys Into Sales
-        </Display>
+        <Title color="white" align="center">
```

Comment:
I would keep this as Display instead of Title

Reply:
This is way too big on mobile thats why I changed

Reply:
alright fixed